Automattic/jetpack
# [RNMobile] Add settings toggle to VideoPress block

## Proposed changes:

* This is a small PR that adds a settings toggle to the VideoPress block in the native apps.
* As per the pattern followed with other media blocks, the toggle only displays when media has been uploaded and when the block is selected.
* Individual settings will be added with future PRs. For now, "Hello world!" placeholders have been added to the settings panel.

### Other information:


## Jetpack product discussion

Designs for block settings can be found here: p9ugOq-3tP-p2 

## Does this pull request change what data or activity we track or use?

No, it doesn't.

## Testing instructions:

* With these branches checked out, open the editor and add the VideoPress block in either app.
* Verify that no settings toggle appears in the block's empty state.
* Upload a video to the block.
* Verify that the new settings toggle appears while the block is selected. Note, there are currently no settings within the settings panel. These will be added with future PRs.
* The toggle should stop displaying when the block _isn't_ selected. 

| Before  | After |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/222424483-5e6957be-5406-43a1-9078-8a3a68355c2c.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/222423568-21464f40-f45a-4e7a-b094-f812dd8e7199.png" width="100%"> |
